axum-extra/src/extract/multipart.rs

```diff
@@ -246,6 +247,58 @@ impl MultipartError {
     fn from_multer(multer: multer::Error) -> Self {
         Self { source: multer }
     }
+
+    /// Get the response body text used for this rejection.
+    pub fn body_text(&self) -> String {
+        self.source.to_string()
+    }
+
+    /// Get the status code used for this rejection.
+    pub fn status(&self) -> http::StatusCode {
+        status_code_from_multer_error(&self.source)
+    }
+}
+
+fn status_code_from_multer_error(err: &multer::Error) -> StatusCode {
+    match err {
+        multer::Error::UnknownField { .. }
+        | multer::Error::IncompleteFieldData { .. }
+        | multer::Error::IncompleteHeaders
+        | multer::Error::ReadHeaderFailed(..)
+        | multer::Error::DecodeHeaderName { .. }
+        | multer::Error::DecodeContentType(..)
+        | multer::Error::NoBoundary
+        | multer::Error::DecodeHeaderValue { .. }
+        | multer::Error::NoMultipart
+        | multer::Error::IncompleteStream => StatusCode::BAD_REQUEST,
+        multer::Error::FieldSizeExceeded { .. } | multer::Error::StreamSizeExceeded { .. } => {
```

Comment:
This kind of error will never thrown, because `axum` does not support constraints of `multer`.

Reply:
But we still need an exhaustive match.